psi4/src/psi4/libfock/cubature.cc

```diff
@@ -53,18 +52,15 @@ using namespace psi;
 
 namespace {
 
-double GetBSRadius(unsigned Z)
-{
+double GetBSRadius(unsigned Z) {
     // Not sure where these numbers come from...
     static const double BSRadii[55] = {
-        1.000,
-        1.001,                                                                                                                 1.012,
-        0.825, 1.408,                                                                       1.485, 1.452, 1.397, 1.342, 1.287, 1.243,
-        1.144, 1.364,                                                                       1.639, 1.716, 1.705, 1.683, 1.639, 1.595,
-        1.485, 1.474, 1.562, 1.562, 1.562, 1.562, 1.562, 1.562, 1.562, 1.562, 1.562, 1.562, 1.650, 1.727, 1.760, 1.771, 1.749, 1.727,
-        1.628, 1.606, 1.639, 1.639, 1.639, 1.639, 1.639, 1.639, 1.639, 1.639, 1.639, 1.639, 1.672, 1.804, 1.881, 1.892, 1.892, 1.881,
+        1.000, 1.001, 1.012, 0.825, 1.408, 1.485, 1.452, 1.397, 1.342, 1.287, 1.243, 1.144, 1.364, 1.639,
+        1.716, 1.705, 1.683, 1.639, 1.595, 1.485, 1.474, 1.562, 1.562, 1.562, 1.562, 1.562, 1.562, 1.562,
+        1.562, 1.562, 1.562, 1.650, 1.727, 1.760, 1.771, 1.749, 1.727, 1.628, 1.606, 1.639, 1.639, 1.639,
+        1.639, 1.639, 1.639, 1.639, 1.639, 1.639, 1.639, 1.672, 1.804, 1.881, 1.892, 1.892, 1.881,
```

Comment:
should unformat this block

Reply:
Good catch; I just blindly ran c-f.  Will have the changes in before PR merge deadline.